flags.go

```diff
@@ -3,9 +3,16 @@ package kingpin
 import (
 	"fmt"
 	"os"
+	"regexp"
 	"strings"
 )
 
+var (
+	envVarValuesSeparator = "\r?\n"
```

Comment:
here we go for windows support - still haven't answered your comment about how to make this configurable though? maybe it can be done later?